plugins/package-managers/node/src/funTest/assets/projects/synthetic/npm/project-with-lockfile/package.json

```diff
@@ -10,8 +10,8 @@
   "author": "The Author <the.author@mail.example> (https://www.the.author.example/index.html)",
   "private": true,
   "dependencies": {
-    "long": "^3.2.0",
     "cheerio": "1.0.0-rc.1",
+    "long": "^3.2.0",
```

Comment:
Please explain in the commit message *why* these are reordered, and to *which* order. I guess to have alphabetical ordering, but why is that important here? Just for readability / consistency, or something else?

Maybe instead of "re-order" simply say "sort" to make things clearer?